tests/integ/test_tf_script_mode.py

```diff
@@ -49,7 +50,7 @@ def test_mnist(sagemaker_session, instance_type):
                            py_version='py3',
                            framework_version=TensorFlow.LATEST_VERSION,
                            metric_definitions=[{'Name': 'train:global_steps', 'Regex': r'global_step\/sec:\s(.*)'}],
-                           base_job_name='test-tf-sm-mnist')
+                           base_job_name=unique_name_from_base('test-tf-sm-mnist'))
```

Comment:
this won't work... need to remove base_job_name, and use unique name as job_name arg to fit function. otherwise the base_job_name may be truncated before standard timestamp is added, and you end up with duplicate names again.

Reply:
oops. i missed this comments. I will make another pr to fix this.